modules/k8s-nmstate-uninstall-operator.adoc

```diff
@@ -0,0 +1,212 @@
+// Module included in the following assemblies:
+//
+// networking/k8s_nmstate/k8s-nmstate-about-the-kubernetes-nmstate-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="k8s-nmstate-uninstall-operator_{context}"]
+= Uninstalling the Kubernetes NMState Operator
+
+You can use the {olm-first} to uninstall the Kubernetes NMState Operator, but by design {olm} does not delete any associated custom resource definitions (CRDs), custom resources (CRs), or API Services. 
+
+Before you uninstall the Kubernetes NMState Operator from the `Subcription` resource that {olm} uses, identify what Kubernetes NMState Operator resources to delete. This identification ensures that you can delete the resources without impacting your running cluster.
+
+.Procedure
+
+. Find `openshift-nmstate` namespace that exists in the `Subcription` resource that {olm} uses by running the following command:
```

Comment:
Do we need this step. Could we just assume the namespace exists?

Reply:
If you use OpenShift with OLM as we instruct people to do, this will be the namespace. No need to go and figure it out